scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
Well for me, `make electron develop` always seems to take ages. And IIRC you said in one of the meetings that you want to optimise the time taken by the CI tests? So wouldn't it be better to just "fix" the optional dependencies sanity check in some way?
(when you say "not needed" above, did you actually mean "not installed"?)

BTW, if this really is a problem, how come we never noticed it before???